sashimi-webapp/src/components/file-manager/Documents.vue

```diff
@@ -23,11 +23,16 @@
           :file="file"
     >
     </file>
+    <template v-if="!docs || (docs.childFileList.length === 0 && docs.childFolderList.length === 0)">
```

Comment:
You may need to handle the case where docs is not a Folder object, in this case, childFileList and childFolderList may not exist